csrc/layer_norm_cuda_kernel.cu

```diff
@@ -893,6 +907,7 @@ void HostApplyRMSNorm(
     const V* gamma)
 {
     auto stream = at::cuda::getCurrentCUDAStream().stream();
+    const int warp_size = at::cuda::getCurrentDeviceProperties()->warpSize;
```

Comment:
We might want to use `at::cuda::warp_size()` instead of `at::cuda::getCurrentDeviceProperties()->warpSize` to be consistent with other places in PyTorch where we need to determine the warp size eg.: https://github.com/pytorch/pytorch/pull/72682/files